v21.2/performance.md

```diff
@@ -8,17 +8,17 @@ docs_area: manage
 
 CockroachDB delivers predictable throughput and latency at all scales on commodity hardware. This page provides an overview of the performance profiles you can expect, based on Cockroach Labs' extensive testing using industry-standard benchmarks like TPC-C and Sysbench.
 
-For instructions to reproduce the TPC-C results listed here, see [Performance Benchmarking with TPC-C](performance-benchmarking-with-tpcc-large.html). If you fail to achieve similar results, there is likely a problem in either the hardware, workload, or test design. Instructions to reproduce Sysbench and KV results are coming soon.
+For instructions to reproduce the TPC-C results listed here, see [Performance Benchmarking with TPC-C](performance-benchmarking-with-tpcc-large.html). If you fail to achieve similar results, there is likely a problem in either the hardware, workload, or test design.
 
 {{site.data.alerts.callout_success}}
-This document is about CockroachDB’s performance on benchmarks. For guidance on tuning real workloads, see [SQL Best Practices](performance-best-practices-overview.html), and for guidance on data location techniques to minimize network latency, see [Topology Patterns](topology-patterns.html).  
+This document is about CockroachDB performance on benchmarks. For guidance on tuning real workloads, see [SQL Best Practices](performance-best-practices-overview.html), and for guidance on data location techniques to minimize network latency, see [Topology Patterns](topology-patterns.html).
 {{site.data.alerts.end}}
 
 ## Scale
 
-TPC-C provides the most realistic and objective measure for OLTP performance at various scale factors, and CockroachDB can process **1.68M tpmC with 140,000 warehouses, resulting in an efficiency score of 95%.** As shown in the chart below, this is a 40% improvement over the results from CockroachDB 19.2.
+TPC-C provides the most realistic and objective measure for OLTP performance at various scale factors. CockroachDB 21.1 can process **1.68M tpmC with 140,000 warehouses, resulting in an efficiency score of 95%.** As shown in the following chart, this is a 40% improvement over the results from CockroachDB 19.2.
```

Comment:
I don't think adding this version number in this PR is a good idea.  These pages go through a lot of vetting every time they are updated.  Style edits are fine but I'm against any content additions.  Especially in a big "style changes" PR like this.  Too easy to miss mistakes. If you want to make non-style content edits I'd rather send you to engineering reviewers

(This may sound extreme but I've spent literally weeks having back-and-forths on some of these pages in the past, it's very easy to introduce small errors)

Reply:
Thanks for the great catch. Totally agree. Not sure why I put that in. Fixed.

Reply:
@rmloveland Now I remember why I put the version in. There's a comparison with CockroachDB 19.2 at the end. It's a weird comparison if one side has a version and the other doesn't. Thoughts?

Reply:
IMO the best thing is to leave this alone until the next content (non-style) edit